handoff.md

```diff
@@ -1,31 +1,77 @@
-# Welcome labs_38 Frontend Devs!
+# Welcome labs_39 Frontend Devs!
 
-##### This file is meant to help clear things up and provide a good starting direction for you because, I think I speak for all of labs_37 when I say,  we know how overwhelming this can be at first.  If you get too stress or have any questions and the answer is not in here, know that you can reach out to any member of the previous cohort on slack for help.  Good luck this month!  
+##### This file is meant to help clear things up for the frontend repo and provide some direction as you start on the project. Be sure to also look at the Trello board user stories as well to know where you can start your work - that’s the best way to begin understanding the codebase. We know how overwhelming this can be at first. If you get stressed or have any questions and the answer is not in here, know that you can reach out to any member of the previous cohort on slack for help. Good luck this month! 
+
+### Product Review
+Feel free to watch our most recent product review meeting where we highlight some of the most recent work we’ve done and the next steps to take. https://www.youtube.com/watch?v=K92fUDY-xMQ&ab_channel=LambdaSchool 
 
```

Comment:
The documentation and comments are well organized. This is a big improvement and will be helpful for the next team.